src/app/app.component.html

```diff
@@ -1,4 +1,4 @@
-<app-confirmation-modal *ngIf="isConfirmModal" [message]='confirmMessage' (close)='confirmed($event)'></app-confirmation-modal>
+<!-- <app-confirmation-modal *ngIf="confirmModalOpen" [message]='confirmMessage' (close)='optionTranslate($event)'></app-confirmation-modal> -->
```

Comment:
@Harshwardhan18 Bro, don't comment this, this is used for other purpose as well

Reply:
@Manzurkds Bro it wasn't used anywhere it was me who added this so just commented it out.

Reply:
@Harshwardhan18 Still you can keep this bro, if you're not setting confirmBoolean, it won't matter if it is there